frontend/sw360-portlet/pom.xml

```diff
@@ -21,47 +21,74 @@
         <jquery.datatables.version>1.10.19</jquery.datatables.version>
         <jquery.datatables.buttons.version>1.5.3</jquery.datatables.buttons.version>
         <jquery.datatables.select.version>1.2.7</jquery.datatables.select.version>
         <jquery.confirm.version>3.3.2</jquery.confirm.version>
         <jquery.treetable.version>3.2.0</jquery.treetable.version>
         <jquery.validation.version>1.19.0</jquery.validation.version>
+
+        <liferay.portal.kernel.version>4.2.5</liferay.portal.kernel.version>
+        <liferay.util.bridges.version>6.0.3</liferay.util.bridges.version>
+        <liferay.util.taglib.version>4.1.1</liferay.util.taglib.version>
+        <liferay.util.java.version>4.0.7</liferay.util.java.version>
+
+        <require.js.version>2.3.6</require.js.version>
+
         <resumable.js.version>1.1.0</resumable.js.version>
-        <context.antiJARLocking>true</context.antiJARLocking>
-        <context.antiResourceLocking>true</context.antiResourceLocking>
-    </properties>
 
-    <artifactId>sw360-portlet</artifactId>
-    <packaging>war</packaging>
+        <artifact.deploy.dir>${liferay.deploy.dir}</artifact.deploy.dir>
+    </properties>
 
     <build>
+	<finalName>org.eclipse.sw360.portlet-${project.version}</finalName>
         <plugins>
             <plugin>
-                <groupId>com.liferay.maven.plugins</groupId>
-                <artifactId>liferay-maven-plugin</artifactId>
-                <version>${liferay.maven.plugin.version}</version>
+                <groupId>com.liferay</groupId>
+                <artifactId>com.liferay.css.builder</artifactId>
+                <version>2.0.2</version>
                 <executions>
                     <execution>
-                        <phase>generate-sources</phase>
+                        <id>default-build-css</id>
+                        <phase>generate-resources</phase>
                         <goals>
-                            <goal>build-css</goal>
+                            <goal>build</goal>
                         </goals>
                     </execution>
                 </executions>
                 <configuration>
-                    <autoDeployDir>${deploy.dir}</autoDeployDir>
-                    <appServerDeployDir>${liferay.app.server.deploy.dir}</appServerDeployDir>
-                    <appServerLibGlobalDir>${liferay.app.server.lib.global.dir}</appServerLibGlobalDir>
-                    <appServerPortalDir>${liferay.app.server.portal.dir}</appServerPortalDir>
-                    <liferayVersion>${liferay.version}</liferayVersion>
-                    <pluginType>portlet</pluginType>
+                    <docrootDirName>src/main/resources/META-INF/resources</docrootDirName>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>biz.aQute.bnd</groupId>
+                <artifactId>bnd-maven-plugin</artifactId>
+                <version>3.5.0</version>
```

Comment:
I noticed that this plugin is used at several pom.xml files (further down below) and I wondered if it would make sense to put the used version also into a top level variable?